src/scripts/worker/worker.js

```diff
@@ -0,0 +1,101 @@
+(function(tiy){
```

Comment:
Should we name this worker specifically for the `hw` module? If we add another worker for some other feature we might not want the confusion.